src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+@import "../utils/mixins";
+@import "../utils/variables";
+
+.calendar {
+  &__page {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendarWidth;
+    margin: 50px auto 0;
+    padding: $padding;
+    gap: $gap;
+  }
+
+  &__day {
+    @include squere();
+    display: flex;
+    border: 1px solid black;
+    transition: 0.5s;
+    background-color: #eee;
```

Comment:
You should specify changeable properties
Don't set transition for changable properties.